tests/e2e/specs/empty-trash-restore-trashed-posts.test.js

```diff
@@ -0,0 +1,71 @@
+import {
+    visitAdminPage,
+    createNewPost,
+    trashAllPosts,
+    publishPost
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Empty Trash', () => {
+
+
+    async function createPost() {
+
+        // Create a Post
+        const title = 'Test Title';
+        await createNewPost( { title } );
+        await publishPost();
+
+    }
+
+
+    afterEach( async () => {
+        await trashAllPosts();
+    } );
+
+    it( 'Empty Trash', async () => {
+
+        await createPost();
+       
+        await visitAdminPage( '/edit.php' );  
+
+        // Move post to trash
+        await page.waitForSelector( '#the-list .type-post' );
+        await page.hover('.row-title');
+        await page.click("a[aria-label='Move “Test Title” to the Trash']");
+
+        // Empty trash 
+        await page.click("a[href='edit.php?post_status=trash&post_type=post']");  
+        const deleteAllButton = await page.waitForSelector('input[value="Empty Trash"]');
+        await deleteAllButton.click();
+        await page.waitForSelector("#message");
```

Comment:
Ideally, we should still make an assertion about the confirmation message here.

Reply:
Hi @kevin940726,

I tried to make an assertion about confirmation message but unfortunately it didn't worked, when I am adding a assertion message the number of deleted post are dynamic(sometime value is 7 sometime 8,9) so I had to use this. :(

Reply:
I think it's because we aren't resetting the state after each test correctly. `trashAllPosts` only puts all active posts into the trash, but doesn't delete them.

We'll need a `deleteAllPosts` util similar to what I've done in
[Playwright](https://github.com/WordPress/gutenberg/pull/38570/files#diff-b0be83d71db6ce709700ada10e1f3cef99d8d59b2c30c11861a96e083e68dc7d) to force delete all posts after each test. For now, I think we can use the `__experimentalRest` api to create one in this file, and replace that with `deleteAllPosts` once we migrate to playwright.

If that's too much work for now, I'm okay if we just add a `// TODO` comment above and leave that to our future selves.

Reply:
Okay @kevin940726, I have added `// TODO` for now.